### DISCUSSION TITLE
Is there a way to link packages without modifying 'package.json'?

### DISCUSSION THREAD

**Discussion by alexcui03** [yarnpkg/berry#2956] [Q&A]
I use Yarn `3.0.0-rc.2` in my project. When I link anothor package to my project, like `yarn link ../<package-name>`, Yarn modifies `package.json` and add `resolution` to it. However, I don't want to add any extra info to `package.json` for the linkage. It there a way to avoid it? Or can it be written to some temporary file in `.yarn` directory?

**Comment by arcanis** [comment]
No, this isn't possible at the moment.
